Sources/AppcuesKit/Appcues.swift

```diff
@@ -12,6 +12,10 @@ import Foundation
 @objc(Appcues)
 public class Appcues: NSObject {
 
+    // internal static, made available to the AppcuesTargetElementTrait
+    @available(iOS 13.0, *)
+    static var elementTargeting: AppcuesElementTargeting = UIKitElementTargeting()
```

Comment:
Open to ideas here. 

Other options considered were somehow passing this through from the `TraitComposer`, using  some mechanism to identify which traits needed access to it (i.e. a protocol or certain type); passing the `Appcues` instance through to traits somehow like we do Actions.

That last one seemed quite invasive in particular, for this special case need. But somehow, the `AppcuesTargetElementTrait` needs access during rendering, and the `UIDebugger` needs access during screen capture.

Reply:
sounds good, I'm on board with using the `static` here, seems appropriate in this case.